src/components/GeoLookupControl/GeoLookupControl.module.scss

```
@@ -0,0 +1,49 @@
+  flex: 1;
+  min-width: 0;
+}
+
+.go-button {
+  flex-shrink: 0;
+}
+
+.error {
+  margin-top: theme.$spacingXSmall;
+  font-size: theme.$textSizeXSmall;
+  // Material red 700; promote to a theme token when another component needs it
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)